src/netex_france/offer.rs

```diff
@@ -448,15 +452,18 @@ impl<'a> OfferExporter<'a> {
         Exporter::generate_id(&order_id, object_type)
     }
 
-    fn generate_location(&self, coord: &'a Coord) -> Result<Element> {
+    fn generate_location(&self, coord: &'a Coord) -> Result<Option<Element>> {
+        if coord.lat == 0.0 && coord.lon == 0.0 {
```

Comment:
```suggestion
       if *coord == Coord::default() {
```

Reply:
Done in 86a310621db97e5c506bc77a9c545e4559a867f9.